docs/howtoguides/enable_anbox.md

```diff
@@ -0,0 +1,45 @@
+# How to enable Anbox Cloud
+
+```{important}
+[Anbox Cloud is supported on 20.04 and 22.04 releases](https://anbox-cloud.io/).
+```
+
+To use Anbox, you will need to enable it directly throught the Ubuntu Pro Client,
+which will install all of the necessary snaps and setup the APT sources needed for
+the service.
+
+```{important}
+The Anbox Cloud service cannot be installed on containers
+```
+
+To enable Anbox Cloud, run:
+
+```console
+$ sudo pro enable anbox-cloud
+```
+
+You should see output like the following, indicating that Anbox Cloud
+was correctly enabled on your system:
+
+```
+One moment, checking your subscription first
+Installing required snaps
+Installing required snap: amc
+Installing required snap: anbox-cloud-appliance
+Installing required snap: lxd
+Updating package lists
+Anbox Cloud enabled
+To finish setting up the Anbox Cloud Appliance, run:
+
+$ sudo anbox-cloud-appliance init
+
+You can accept the default answers if you do not have any specific
+configuration changes.
+For more information, see https://anbox-cloud.io/docs/tut/installing-appliance
+```
+
```

Comment:
Do we need to include anything about disabling/removing it?

Reply:
I  have added that section